plugins/aea-crypto-cosmos/aea_crypto_cosmos/cosmos.py

```diff
@@ -699,13 +723,17 @@ def get_handle_transaction(
         )
         if account_number is None or sequence is None:
             return None
+        if self.cosmwasm_versions != DEFAULT_COSMWASM_VERSIONS and amount == 0:
```

Comment:
I think it passing an empty list should work also for an older version of CosmWasm. But we would have to test it.